lib/bolt/inventory/inventory2.rb

```diff
@@ -122,8 +122,15 @@ def update_target(target)
           data['config'] = {}
         end
 
-        unless data['config']['transport']
-          data['config']['transport'] = 'local' if target.name == 'localhost'
+        if target.name == 'localhost'
+          unless data['config']['transport']
+            data['config']['transport'] = 'local'
+          end
+          t = data['config']['transport']
```

Comment:
Should `t` just be hard coded to `local`? We only want to set interpreter for `local`.

Reply:
In a convo with Alex we decided to apply this to localhost on all transports, but I think given comments above we're reversing that.

Reply:
Oh, that is what alex said. Sorry, didn't see that review first.

Reply:
Np - I also assumed it was only on the local transport, and you totally weren't in the convo where we decided otherwise! I should've updated the ticket.